### PR TITLE
New TH-TIN record fixes #377

### LINKED ISSUE #377: PROPOSAL: new list for Thailand
## Proposal

**The name of the list (or the organization that manages the list)**
Tax Identification Number

**A suggested code for the list (optional)**
TIN

**A short description of the list**
The usual tax number, used both for natural and legal persons.

**The country or countries that the list covers**
Thailand

**The legal form or organizations that the list covers**
All

**Any specific sectors that the list covers**
All

**A URL for information on the list, and a URL for looking up identifiers (if available)**
The Thai government website has a document mentioning 10 digits, but that apparently is old, and the number is now 13 digits, since 2012.
Government: http://www.rd.go.th/publish/21987.0.html
Blog by external source mentioning the change in law: https://blogs.exact.com/products/2012/11/new-thai-tax-identification-number-introduced
I can't find any online source for looking up identifiers but that may be a language issue. The government site is here: http://vsreg.rd.go.th/

**Any information on available open data copies of this list**
None I'm aware of.

**One or more examples of identifier from this list, and how you found them**
099-4-00036896-8 for Plan International, Regional Office for Asia (not to be confused with Plan International Thailand). See next comment.

### PR DESCRIPTION
**Pull request by kd-ods**
I couldn't find a simple UI for searching for or finding TINs, but there is a SOAP API providing limited validation of TINs.

### CODE REVIEW

File: lists/th/th-tin.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": {
+    "en": "Thai Revenue Department - Tax Identification Number",
+    "local": "หมายเลขประจำตัวผู้เสียภาษี"
+  },
+  "url": "https://www.rd.go.th/publish/272.0.html",
+  "description": {
+    "en": "This is the standard tax number issued to legal entities that pay taxes or that pay income to employees. NGOs require one, too. The procedure is here: http://www.rd.go.th/publish/21987.0.html The government site still mentions a 10-digit number but the law changed in 2012 and the TIN is now 13 digits long (see: https://msnagroup.com/new-thai-tax-id-number). The TIN should not be confused with the PIN, for individuals."
```
Comment (siwhitehouse) [inline lists/th/th-tin.json:8]: Individuals can be issued with a TIN if they are a foreigner or if they want to register for Value Added Tax or Specific Business Tax, according to http://www.rd.go.th/publish/21987.0.html. 

Also, foreign companies (and NGOs) operating in Thailand require a TIN and so possession of a TIN does not necessarily equate to an entity being on the Thai company register. That's my reading of it, anyway.

Are these points worth adding?

**Review by siwhitehouse — COMMENTED** [review]
I have a couple of considerations for the description. Other than that it all looks good.

File: lists/th/th-tin.json (hunk 1 above)
Comment (kd-ods) [reply lists/th/th-tin.json]: I've edited to include those bits.

Comment (siwhitehouse) [reply lists/th/th-tin.json]: Thanks @kd-ods - this looks good to merge now

**Review by siwhitehouse — APPROVED** [review]

**Pull request merged**